tests/unit/test_client.py

```diff
@@ -167,12 +166,33 @@ def testRunSearchRequest(self):
             self.assertEqual(result[1].id, "refB")
 
             # verify requests.post called correctly
+            httpMethod = 'POST'
             url = "http://example.com/referencesets/search"
-            jsonString = protocolRequest.toJsonString()
+            data = protocolRequest.toJsonString()
             headers = {"Content-type": "application/json"}
-            httpMethod = 'POST'
-            mockPost.assert_called_twice_with(
-                httpMethod, url, jsonString, headers=headers, verify=False)
+            params = {u'key': u'KEY'}
```

Comment:
Sure.  FYI, these lines needed to be changed after I did a library update.  Something with the interaction of requests / mock / something else?